code/modules/mob/living/living.dm

```diff
@@ -1105,3 +1105,11 @@ default behaviour is:
 /mob/living/get_speech_bubble_state_modifier()
 	return isSynthetic() ? "synth" : ..()
 
+/mob/living/is_on_special_ability_cooldown()
+	return world.time < next_special_ability
+
+/mob/living/set_special_ability_cooldown(var/amt)
+	next_special_ability = max(next_special_ability, world.time+amt)
+
+/mob/living/get_time_until_next_special_ability()
```

Comment:
```suggestion
/mob/living/get_seconds_until_next_special_ability()
```
`get_time` is a bit unclear since as far as I can tell this is returning seconds, not deciseconds? Wouldn't want someone to use it expecting deciseconds. Non-blocking.